wheatley/row_generation/complib_composition_generator.py

```diff
@@ -188,7 +188,7 @@ def summary_string(self) -> str:
     def _gen_row_and_calls(self, _previous_row: Row, _stroke: Stroke, index: int) -> Tuple[Row, List[str]]:
         if index < len(self.loaded_rows):
             return self.loaded_rows[index]
-        return (self.rounds(), [])
+        return (self.start_row, [])
```

Comment:
Is this change actually needed, since `start_row` should always be rounds for `ComplibCompositionGenerator`.  I wonder if it would be better to add a sanity check here to make sure that the code doesn't break silently.  Something like:
```python
assert self.start_row == self.rounds()
return (self.start_row, [])
```

Reply:
I'm not sold on the benefit of the assert over directly `return (rounds(self.stage), [])`

Reply:
Agreed, I originally took the rounds() property off of row_generator but that was before the original thread around compositions.

I have just added it back and returned this to it's original state - seeing as we always want to start from rounds anyway